internal/ciphersuite/aes_ccm.go

```diff
@@ -103,11 +103,11 @@ func (c *AesCcm) Encrypt(pkt *recordlayer.RecordLayer, raw []byte) ([]byte, erro
 }
 
 // Decrypt decrypts a single TLS RecordLayer
-func (c *AesCcm) Decrypt(raw []byte) ([]byte, error) {
+func (c *AesCcm) Decrypt(h recordlayer.Header, raw []byte) ([]byte, error) {
```

Comment:
If we don't want to break this API, we could define separate `DecryptWithHeader` methods.

Reply:
I think it is fine breaking them!

Anyone using these APIs will understand/be able to adapt quickly I bet

Reply:
Nobody can use these APIs outside of the DTLS library as it's in the `internal/` package. The only usages are within this library, so you can break it to your hearts' content.

Reply:
@daenney good point -- I intended to comment on `pkg/crypto/ciphersuite`, which has similar changes.